src/shared/dissect-image.c

```diff
@@ -1515,7 +1521,7 @@ int dissected_image_mount_and_warn(DissectedImage *m, const char *where, uid_t u
         if (r == -ENXIO)
                 return log_error_errno(r, "Not root file system found in image.");
         if (r == -EMEDIUMTYPE)
-                return log_error_errno(r, "No suitable os-release file in image found.");
+                return log_error_errno(r, "No suitable os-release/extension-release.d/extension-release.<name> file in image found.");
```

Comment:
i'd not make this needlessly confusing. i.e. just say "os-release/extension-release file"